go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/dop251/goja v0.0.0-20200721192441-a695b0cdd498
 	github.com/edsrzf/mmap-go v1.0.0
 	github.com/fatih/color v1.7.0
+	github.com/fjl/gencodec v0.0.0-20191126094850-e283372f291f // indirect
```

Comment:
I think this was added indirectly because you did a go get gencodec. Please revert the deps.